tests/testsuite/test.rs

```diff
@@ -48,6 +49,37 @@ fn cargo_test_simple() {
         .run();
 }
 
+#[cargo_test]
+// download a known broken doctests code base and run cargo test, should fail
+fn cargo_test_tensorflow_not_failing_due_to_shared_library() {
+    use git2::build::RepoBuilder;
+
+    // git repo for cargo project containing doctests that require shared libraries
+    let url = "https://github.com/tensorflow/rust";
```

Comment:
Note that we don't rely on network resources in the tests

Reply:
This was created to reproduce only. I'm hoping you can help me get it shaped into the cargo way.

Reply:
I'm trying to id a minimal test for this, but it's taking longer than I want, so this was a shortcut to open discussion.

Reply:
removing this. only needed for validation.